packages/web/src/brandkit/Logo.tsx

```diff
@@ -44,15 +44,13 @@ export default React.memo(function Logo() {
 })
 
 const Overview = withNamespaces(NameSpaces.brand)(
-  withScreenSize<I18nProps>(function _Overview({ t, screen }: I18nProps & ScreenProps) {
+  withScreenSize(function _Overview({ t, screen }: I18nProps & ScreenProps) {
     const glyphAreaStyle = screen === ScreenSizes.DESKTOP ? styles.pilar : styles.square
     return (
       <View style={styles.container}>
         <View style={styles.gap}>
           <H1 style={standardStyles.elementalMarginBottom}>{t('logo.title')}</H1>
-          <Text style={[fonts.p, standardStyles.elementalMarginBottom]}>
-            {t('logo.overviewCopy')}
-          </Text>
+          <H4 style={[standardStyles.elementalMarginBottom]}>{t('logo.overviewCopy')}</H4>
```

Comment:
Nit: wrapping styles in arrays causes unnecessary re-renders. Not a big deal but there's no reason for an array here